libs/csm/rest/csm_rest_iamuser.py

```diff
@@ -558,7 +558,7 @@ def remove_user_caps_rgw(self, uid, payload: dict):
         return response
 
     @RestTestLib.authenticate_and_login
-    def delete_iam_user_rgw(self, uid, header, purge_data=False):
+    def delete_iam_user_rgw(self, uid, header, purge_data=None):
```

Comment:
purge_data=False was fine.

Reply:
all options are getting ignored with delete user

> [Yesterday 1:22 PM] Rajesh Pratap Singh Chouhan
Rohit are we ignoring all option & "purge_data" in delete REST api with this changes ?i.e. earlier we use to get error incase we gave {"purge_data": abc} or {"abc": abc} with the API..but now we are not considering anything at all so not getting any error for this ? or it's a bug ??cc Pranay Pawan

> [Yesterday 1:31 PM] Pranay Kumar
I think no as per standard ... All extra inputs which are not required is being ignored from all the APIs.....if any api is not ignoring and returning error then it should be a defect ....but I will let pawan confirm this
 like 2

> [Yesterday 1:33 PM] Rohit Kolapkar
yes, We are ignoring all extra inputs.

so if we give {"purge_data": true} or {"purge_data": false} or even  {"purge_data": abc}, all will be ignored

Reply:
ok